components/raftstore/src/router.rs

```diff
@@ -33,7 +37,9 @@ where
         &self,
         region_id: u64,
         msg: SignificantMsg<EK::Snapshot>,
-    ) -> RaftStoreResult<()>;
+    ) -> RaftStoreResult<()> {
+        <Self as SignificantRouter<EK>>::send(self, region_id, msg)
```

Comment:
How about removing `significant_send` and renaming `send` to `significant_send`?

Reply:
`send_casual_msg` and `send_store_msg` does it in the same manner, better to keep it consistent.

Reply:
Now I think keeping them different name may be better. As if a trait replace with another accidentally, it will report error instead of succeed silently.